homeassistant/helpers/entity.py

```diff
@@ -657,6 +674,15 @@ def translation_key(self) -> str | None:
             return self.entity_description.translation_key
         return None
 
+    @property
+    def translation_placeholders(self) -> Mapping[str, str] | None:
```

Comment:
We should probably make this a `cached_property` and mark it `Final` to discourage changing the placeholders

Reply:
Changed to a `cached_property`